### ISSUE TITLE
OpenGL regression in Burnout 3 (draw distance is black)

### ISSUE THREAD

**Issue by bryc** [PCSX2/pcsx2#2736]
***PCSX2 version:***
v1.5.0-dev-2714

***PCSX2 options:***
All defaults.

***Plugins used:***
All defaults. 

***Description of the issue:***
In the OpenGL Hardware plugin, there is a new issue where geometry in the distance is completely black. 

Here is a screenshot: 
![image](https://user-images.githubusercontent.com/1408749/49559777-eba94500-f8dd-11e8-9726-d6bc0b5d7a04.png)

***How to reproduce the issue:***
It is pretty evident in any race. That race in the screenshot is Silver Lake. 

***Last known version to work:***
Seems to work fine in dev-2484.

***PC specifications:***
Tested on Windows 7/i7-3770/GTX 1050Ti and Windows 10/i5-7200U/GT 960M, both affected

**Comment by MrCK1** [comment]
Did you try anything else between 2484 and 2714, or is 2484 the first build that works?

**Comment by bryc** [comment]
2484 is just the latest I have on hand - these are pretty much stock builds in separate folders from the buildbot so i don't mess with the plugins.

**Comment by MrCK1** [comment]
OK then can you try to narrow down further what build caused the issue?

**Comment by bryc** [comment]
I tried the earliest build on the buildbot - `v1.5.0-dev-2695-g193bda164` (NOV 23), and it happens there too. So that would be a commit between August 12 - November 23.

**Comment by lightningterror** [comment]
Regression was introduced in 2670.

It will be fixed once https://github.com/PCSX2/pcsx2/pull/2731 is merged.
